modules/op-resolver-hub-config.adoc

```diff
@@ -33,9 +33,20 @@ spec:
       default-artifact-hub-pipeline-catalog: tekton-catalog-pipelines # <3>
       defailt-kind: pipeline # <4>
       default-type: tekton # <5>
+      tekton-hub-api: "https://my-custom-tekton-hub.example.com" # <6>
+      artifact-hub-api: "https://my-custom-artifact-hub.example.com" # <7>
 ----
 <1> The default {tekton-hub} catalog for pulling a resource.
 <2> The default {artifact-hub} catalog for pulling a task resource.
 <3> The default {artifact-hub} catalog for pulling a pipeline resource.
 <4> The default object kind for references.
 <5> The default hub for pulling a resource, either `artifact` for {artifact-hub} or `tekton` for {tekton-hub}.
+<6> The {tekton-hub} API used, if the `default-type` option is set to `tekton`.
```

Comment:
I think this one and the next one should be marked as optional more clearly.

Reply:
Hello @mramendi I will wait for the answer from SME regarding comment above about the Tekton Hub API value. But the information about having to set the `tekton-hub-api` value is in the supporting docs. If that will be confirmed then the `tekton-hub-api` value is not optional.

Reply:
Note to the previous comment: I had some a minor issue with GitHub and did not see the responses right away, but the response is already there, therefore the the `tekton-hub-api` value is not optional.

Reply:
Agreed - I did not know this at the time I wrote the comment. "Optional" applies only to the artifact hub setting (which in fact is normally *not* needed)